source/guide_littlelinkcustom.rst

```diff
@@ -0,0 +1,110 @@
+.. highlight:: console
+
+.. author:: Jan Klomp <https://klomp.de>
+
+.. categorize your guide! refer to the current list of tags: https://lab.uberspace.de/tags
+.. tag:: lang-php
+.. tag:: web
+.. tag:: self-hosting
+.. tag:: analytics
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/littlelinkcustom.svg
+      :align: center
+
+##########
+LittleLink Custom
+##########
+
+.. tag_list::
+
+LittleLinkCustom_ is a highly customizable link sharing and landing page platform with an easy to use interface and acts as a free opensource alternative to Linktree, Shorby, bio.fm etc.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+    * :manual:`PHP <lang-php>`
+    * :manual:`domains <web-domains>`
+
+License
+=======
+
+*LittleLink Custom* is licensed under the GNU General Public License v3.0. All relevant legal information can be found here:
+
+  * https://github.com/JulianPrieber/littlelink-custom/blob/main/LICENSE
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 8.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '8.1'
+ [isabell@stardust ~]$
+
+If you want to use *LittleLink Custom* with your own domain you need to setup your :manual:`domains <web-domains>` first:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+``cd`` to your :manual:`document root <web-documentroot>`, then download the latest release of *LittleLink Custom* and extract it:
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ wget https://github.com/JulianPrieber/littlelink-custom/releases/latest/download/littlelink-custom.zip
+ ...
+ [isabell@stardust html]$ unzip littlelink-custom.zip
+ ...
+ [isabell@stardust ~]$
+
+Move the extracted files and directories to the document root, then delete the empty source directory, the downloaded zip file and the ``nocontent.html`` file:
```

Comment:
proposing easier way for download->unzip and creating structure:

```
$ cd /var/www/virtual/$USER/ #one level above html
$ wget https://github.com/JulianPrieber/littlelink-custom/releases/latest/download/littlelink-custom.zip
$ unzip littlelink-custom.zip littlelink-custom/* #get only the files from the subfolder
$ rm html/noconten.html; rmdir html/ #remove the default html folder in a safe way
$ rm littlelink-custom.zip
$ mv littlelink-custom html
```

by the way, you dont need to put an empty `[isabell@stardust ~]$` this is just thought to be at the end of a codeblock